ui/src/utils/currencyPair.tsx

```diff
@@ -147,13 +147,13 @@ export function CurrencyPairProvider({
 
     setMintAddressA(
       tokens.find((t) => t.tokenSymbol === defaultBase)?.mintAddress ||
-        (isValidAddress(defaultBase) ? defaultBase : "") ||
-        ""
+      (isValidAddress(defaultBase) ? defaultBase : "") ||
+      ""
     );
     setMintAddressB(
       tokens.find((t) => t.tokenSymbol === defaultQuote)?.mintAddress ||
-        (isValidAddress(defaultQuote) ? defaultQuote : "") ||
-        ""
+      (isValidAddress(defaultQuote) ? defaultQuote : "") ||
+      ""
```

Comment:
did this file change at all, or are these only formatting changes? if it's only formatting it's better to not change it at all, so it won't make it harder to merge new changes from the original serum swap repo, in case they develop new features.